search_api_doc.rst

```diff
@@ -0,0 +1,225 @@
+Search API
+==========
+
+Method::
+   
+   GET http://archive.cnx.org/search?q=
+
+Usage::
+   
+   QUERY [author:AUTHOR] [authorID:ID] [title:TITLE] [subject:SUBJECT] ...
+         [keyword:KEY] [type:TYPE] [language:LANG] [pubYear:YEAR] [sort:SORT]
+
+Brakets denote an optional parameter.
+
+
+Parameters:
+
++---------+---------+---------------------------------+
+| Field   | Type    | Description                     |
++---------+---------+---------------------------------+
+| QUERY   | String  | General search string           |
++---------+---------+---------------------------------+
+| AUTHOR  |         | Search by author contact info.  |
++---------+---------+---------------------------------+
+| ID      |         | Search by a specific author ID  |
```

Comment:
Ah ok, I see what you are saying. Looks fine, now that I understand where you were going with it.